keras_cv/layers/preprocessing/rand_augment.py

```diff
@@ -76,7 +79,7 @@ def __init__(
         augmentations_per_image=3,
         magnitude=0.5,
         magnitude_stddev=0.15,
-        rate=1.0,
+        rate=10 / 11,
```

Comment:
1/11 or 10/11?

Reply:
10/11 because this is the rate we apply augmentations at.

Reply:
Ha - no worries now I see you were looking at the docstring.  I fixed it - thanks Kartik!

Reply:
got confused from doctoring, cool